File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/chat_completion_metadata.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatCompletionMetadata:
    """聊天完成元数据"""
    actions: List[str] = None
    agent: Dict[str, Any] = None
    parent_message_id: int = None
    parent_message_uuid: str = None

File: sdkwork-backend-sdk-typescript/src/types/chat-completion-metadata.ts
/** 聊天完成元数据 */
export interface ChatCompletionMetadata {
  /** 可用操作列表 */
  actions?: string[];
  /** 代理相关信息 */
  agent?: Record<string, Record<string, unknown>>;
  /** 父级聊天ID */
  parent_message_id?: number;
  /** 父级聊天UUID */
  parent_message_uuid?: string;
}
